README.md

```diff
@@ -596,6 +596,12 @@ For more information, see [CONTRIBUTING](https://github.com/pyllyukko/user.js/bl
 
 --------------------------------------------------------------------------
 
+## License
```

Comment:
This is already quite evident at least when browsing the README through GitHub:

![license](https://cloud.githubusercontent.com/assets/1058704/24071421/f9d37396-0bd9-11e7-8729-6cceba0c4222.png)